kasa/smart/modules/energy.py

```diff
@@ -5,56 +5,22 @@
 from typing import TYPE_CHECKING
 
 from ...emeterstatus import EmeterStatus
-from ...feature import Feature
+from ...exceptions import KasaException
+from ...interfaces.energy import Energy as EnergyInterface
 from ..smartmodule import SmartModule
 
 if TYPE_CHECKING:
-    from ..smartdevice import SmartDevice
+    pass
```

Comment:
Unnecessary.

Reply:
I've finally realised that ruff keeps doing this to me when it deems a type checking import unnecessary 😳 .  Will fix.

Reply:
Yeah, happens often when converting from __init__ to use initialize_features, as that's usually the only place where it's used :-)

Reply:
Removed